pcsx2-qt/Settings/GraphicsSettingsWidget.cpp

```diff
@@ -390,6 +402,17 @@ GraphicsSettingsWidget::GraphicsSettingsWidget(SettingsWindow* dialog, QWidget*
 		onVideoCaptureAutoResolutionChanged();
 		onEnableAudioCaptureChanged();
 		onEnableAudioCaptureArgumentsChanged();
+
+		if (m_dialog->isPerGameSettings())
+		{
+			m_ui.videoDumpDirectory->deleteLater();
```

Comment:
You'll need to remove the widget from the layout here too.

Reply:
Done